examples/experimental/dagster-airlift/dagster_airlift/core/sensor/sensor_builder.py

```diff
@@ -125,10 +125,17 @@ def airflow_dag_sensor(context: SensorEvaluationContext) -> SensorResult:
                 end_date_lte=None,
                 dag_query_offset=0,
             )
+        transformed_events = (
+            event_transformation_fn(all_events) if event_transformation_fn else all_events
+        )
+
         context.update_cursor(serialize_value(new_cursor))
+        sorted_events = sorted_asset_events(transformed_events, repository_def)
 
         return SensorResult(
-            asset_events=sorted_asset_events(all_materializations, repository_def),
+            asset_events=event_transformation_fn(sorted_events)
+            if event_transformation_fn
```

Comment:
did you mean to call event_transformation_fn twice?

Also just in terms of readability and navigability I think we should directly call the "default" function here so you can jump right to it in the IDE?